actionpack/lib/action_dispatch/railtie.rb

```diff
@@ -13,12 +13,12 @@ class Railtie < Rails::Railtie # :nodoc:
     config.action_dispatch.default_charset = nil
     config.action_dispatch.rack_cache = false
     config.action_dispatch.http_auth_salt = "http authentication"
-    config.action_dispatch.signed_cookie_salt = "signed cookie"
-    config.action_dispatch.encrypted_cookie_salt = "encrypted cookie"
-    config.action_dispatch.encrypted_signed_cookie_salt = "signed encrypted cookie"
     config.action_dispatch.use_authenticated_cookie_encryption = false
     config.action_dispatch.perform_deep_munge = true
 
+    config.action_dispatch.legacy_encryptor_schemes = []
+    config.action_dispatch.legacy_verifier_schemes  = []
```

Comment:
I agree with these wording changes.

The word "scheme" is unclear and could refer to several things really. It's probably best to just avoid that term altogether.

Removing the "legacy" wording is a good idea as well since this feature is more than just legacy support. Your configuration above is really nice and clean! I can rework my changes to introduce this style of configuration for this feature.